cpp-client/deephaven/examples/concurrent_client/main.cc

```diff
@@ -32,9 +34,9 @@ void ThreadFun(std::shared_ptr<Client> clientp, const std::size_t ti) {
 }
 
 void Run(std::shared_ptr<Client> clientp, const std::size_t nthreads) {
-  std::thread threads[nthreads];
+  std::vector<std::thread> threads(nthreads);
```

Comment:
Unfortunately this won't do what you want.  As written this will populate the vector with nthread slots, and then below you will add nthreads more slots.

I believe you intended

```
 std::vector<std::thread> threads;
 threads.reserve(nthreads);
```